rpc/handler.go

```diff
@@ -571,7 +574,20 @@ func (h *handler) handleCallMsg(ctx *callProc, msg *jsonrpcMessage) *jsonrpcMess
 		if resp.Error != nil {
 			ctx = append(ctx, "err", resp.Error.Message)
```

Comment:
(optional) I appreciate that this is pre-existing code, but the `ctx` variable is idiomatically reserved for a `context.Context`. Seeing it being appended to as a slice really confused me; it's also shadowing a method argument. Perhaps `logCtx` or (my preference because this isn't a context) `logData`?

Reply:
yup it's definetly confusing, changed to logCtx

Reply:
I did not want to drift too much away from `ctx` (dunno why it's named this way though)